lib/matplotlib/tests/test_subplots.py

```diff
@@ -173,3 +173,15 @@ def test_dont_mutate_kwargs():
                            gridspec_kw=gridspec_kw)
     assert subplot_kw == {'sharex': 'all'}
     assert gridspec_kw == {'width_ratios': [1, 2]}
+
+
+def test_subplot_theta_min_max():
+    with pytest.raises(ValueError, match="Cannot pass angle range > 2 pi"):
+        ax = plt.subplot(111, projection='polar')
+        ax.set_thetalim(thetamin=800, thetamax=400)
```

Comment:
Please add an additional test that 2 pi < degree-range < 360 still works and we're not accidentally cutting these to 2pi as well.

Reply:
yes, I agree that should be one of the test cases. Will make the changes

Reply:
also two more tests that meet this criteria in the new commit